### DISCUSSION TITLE
Welcome to mouseVRheadset Discussions!

### DISCUSSION THREAD

**Discussion by misaacson01** [sn-lab/MouseGoggles#1] [Announcements]
## Welcome!
  Feel free to use the discussions page for more informal, broader questions and comments about the project:
  * Ask questions about the design and functionality of the system
  * Suggest ideas for future development
  * Share your project/use case

**Comment by ranczlab** [comment]
This is really cool - and super useful. We will have a go building it. 
Do you have an estimate of how much in focus the image is on the retina?

**Reply by misaacson01** [reply]
Great! Just FYI we're testing out some slight adjustments on the 3D print files so there will be new versions uploaded sometime this week.

As for the focus on the retina, we _estimate_ that points should be focused much smaller than mice are able to resolve, though we are pursuing some different ways right now to try and measure exactly/directly the amount of spread on the retina. So, we don't have a great answer just yet, but we hope to soon.

**Comment by ranczlab** [comment]
Sounds good! 

I just noticed the mouse eye physiology is already an open issue, I will post there.
